sdk/aqueduct/artifacts/table_artifact.py

```diff
@@ -128,6 +129,9 @@ def head(self, n: int = 5, parameters: Optional[Dict[str, Any]] = None) -> pd.Da
             A dataframe containing the table contents of this artifact.
         """
         df = self.get(parameters=parameters)
+        if df is None:
+            return None
```

Comment:
nit: slightly more python for `return df.head(n) if df else None?

Reply:
oh actually I do this intentionally as I believe `if df` will be `False` if df is empty. But we can shrink that to one line though